catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -601,27 +602,27 @@ private UpdateRequest populateMetacards(UpdateRequest updateRequest) throws Inge
         final String attributeName = updateRequest.getAttributeName();
 
         QueryRequestImpl queryRequest = createQueryRequest(updateRequest);
-        QueryResponse query;
+        QueryResponse queryResponse;
         try {
-            query = queryOperations.doQuery(queryRequest,
+            queryResponse = queryOperations.doQuery(queryRequest,
                     frameworkProperties.getFederationStrategy());
         } catch (FederationException e) {
             LOGGER.debug("Unable to complete query for updated metacards.", e);
             throw new IngestException("Exception during runtime while performing update");
         }
 
-        if (!foundAllUpdateRequestMetacards(updateRequest, query)) {
-            logFailedQueryInfo(updateRequest, query);
+        if (!foundAllUpdateRequestMetacards(updateRequest, queryResponse)) {
+            logFailedQueryInfo(updateRequest, queryResponse);
             throw new IngestException("Could not find all metacards specified in request");
         }
 
-        updateRequest = rewriteRequestToAvoidHistoryConflicts(updateRequest, query);
+        updateRequest = rewriteRequestToAvoidHistoryConflicts(updateRequest, queryResponse);
 
-        HashMap<String, Metacard> metacardMap = new HashMap<>(query.getResults()
+        HashMap<String, Metacard> metacardMap = new HashMap<>(queryResponse.getResults()
```

Comment:
❓ Wonder if we shouldn't add a comment here explaining why we're using the metacard ID instead of the attribute name like we used to.
❗️ We should also add a test for this if possible.